llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -931,6 +931,10 @@ def FeatureStdExtSmepmp
     : RISCVExtension<"smepmp", 1, 0,
                      "'Smepmp' (Enhanced Physical Memory Protection)">;
 
+def FeatureStdExtSmrnmi
+    : RISCVExtension<"smrnmi", 1, 0,
+                     "'Smrnmi' (Extension for Resumable Non-Maskable Interrupts)">;
```

Comment:
I don't think we usually write "Extension for" in the description.

Reply:
Okay, I have deleted the relevant content.